rust/Makefile

```diff
@@ -370,6 +370,7 @@ rust-analyzer:
 
 $(obj)/core.o: private skip_clippy = 1
 $(obj)/core.o: private skip_flags = -Dunreachable_pub
+$(obj)/core.o: private rustc_objcopy = --redefine-sym __aeabi_uldivmod=rust_aeabi_uldivmod
```

Comment:
Are you sure those calls would stay limited to libcore and can't be inlined into other rust code, even with LTO enabled?

Reply:
That's a good point. I am actually not sure if this works with LTO object files.

Reply:
Just checked, LTO object files aren't even ELF files, so objcopy doesn't work on it. However currently we don't have LTO support for in-kernel Rust code and kernel LTO is not supported for 32-bit ARM either.

Reply:
Also, when LTO is enabled the division will remain as division and will not generate compiler-rt calls -- so this hack definitely won't work.

Reply:
I don't know if any libcore method that uses it is marked as `#[inline]` and would thus get inlined even without LTO.